test/Makefile

```diff
@@ -32,16 +34,16 @@ all: $(FFF_TEST_CPP_TARGET) $(FFF_TEST_C_TARGET) $(FFF_TEST_GLOBAL_C_TARGET) $(F
 $(BUILD_DIR)/%.o: %.cpp
 	@echo 'Building file: $<'
 	@echo 'Invoking: GCC C++ Compiler'
-	g++ -I../ -O0 -g3 -Wall -DGTEST_USE_OWN_TR1_TUPLE=1 -c -o "$@" "$<"
+	g++ $(INCLUDES) -O0 -g3 -Wall -DGTEST_USE_OWN_TR1_TUPLE=1 -c -o "$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
 
 $(BUILD_DIR)/%.o: %.c
 	@echo 'Building file: $<'
 	@echo 'Invoking: GCC C Compiler'
-	gcc -I../ -O0 -g3 -Wall -std=c99 -c -o "$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
+	gcc -v $(INCLUDES) -O0 -g3 -Wall -std=c99 -c -o "$@" "$<"
```

Comment:
I wonder if this should be moved two lines up?

Reply:
Thanks for catching that, not sure how that got away from me.